src/main/resources/resources.properties

```diff
@@ -125,7 +125,12 @@ ui.authenticationView.noEmptyToken=The token cannot be empty
 ui.authenticationView.tokenLink=What is my token?
 
 ### Misc
-ui.aboutDialog.description=<html>The A+ Courses plugin supports the educational use of IntelliJ (and its Scala plugin) in<br>programming courses that rely on the A+ learning management system, which has been<br>developed at Aalto University. The plugin accesses programming assignments<br>and automated grading services provided by A+ and otherwise enhances<br>the student experience.</html>
-ui.aboutDialog.title=About the A+ Courses Plugin
+ui.aboutDialog.description=<html>The A+ Courses plugin supports the educational use of IntelliJ (and its Scala plugin)<br>in programming courses that rely on the A+ learning management system, which<br>has been developed at Aalto University.<br><br>The plugin accesses programming assignments and automated grading services<br>provided by A+ and otherwise enhances the student experience.</html>
+ui.aboutDialog.title=A+ Courses
 ui.aboutDialog.website=A+ Courses Plugin website
 ui.localHistory.submission.tag={0}, {1}, Submission {2}
+ui.aboutDialog.version=<html><b>Version: {0}</b></html>
+ui.aboutDialog.authors=<html>Developed by the department of Computer Science at Aalto University.</html>
+ui.aboutDialog.APlusWebsite=A+ website
+ui.aboutDialog.GithubWebsite=A+ Courses Plugin Github
+ui.aboutDialog.attributes=<html><div style='font-size:8px'>3<sup>rd</sup> Party Attributions:<ul><li><a href="https://commons.apache.org/proper/commons-io/">Apache Commons IO</a></li><li><a href="https://plugins.jetbrains.com/plugin/1347-scala">IntelliJ Scala Plugin</a></li><li><a href="https://www.json.org/json-en.html">json.org</a></li><li><a href="https://www.scala-lang.org/api/current/">Scala Standard Library 2.13.4</a></li><li><a href="https://github.com/srikanth-lingala/zip4j">zip4j</a></li></ul></div></html>
```

Comment:
as the lines' lengths are growing longer and longer, I suggest we start breaking lines. How about using the checkstyle's length of:
```xml
  <module name="LineLength">
            <property name="max" value="100"/>
            ...
   </module>
```
❓❓❓

Reply:
That would require change in properties format (XML or [mprops](https://github.com/mprops/mprops-java) or something). This format doesn't support multiline values.